lms/djangoapps/verify_student/services.py

```diff
@@ -177,22 +178,22 @@ def user_status(cls, user):
             'verification_expiry': '',
         }
 
-        # We need to check the user's most recent attempt.
-        try:
-            photo_id_verifications = SoftwareSecurePhotoVerification.objects.filter(user=user).order_by('-updated_at')
-            sso_id_verifications = SSOVerification.objects.filter(user=user).order_by('-updated_at')
-            manual_id_verifications = ManualVerification.objects.filter(user=user).order_by('-updated_at')
-
-            attempt = most_recent_verification(
-                photo_id_verifications,
-                sso_id_verifications,
-                manual_id_verifications,
-                'updated_at'
```

Comment:
Correct me if I'm wrong here but isn't `verifications` we're iterating over here not strictly in order of "latest" first? Won't all (no-expired) SoftwareSecurePhotoVerifications be evaluated before SSOVerifications?